rbac/management/querysets.py

```diff
@@ -95,9 +95,12 @@ def get_role_queryset(request):
         if username != identity_username and not request.user.admin:
             return Role.objects.none()
         else:
-            return get_object_principal_queryset(request, PRINCIPAL_SCOPE, Role,
-                                                 **{'prefetch_lookups_for_ids': 'access',
-                                                    'prefetch_lookups_for_groups': 'policies__roles'})
+            queryset = get_object_principal_queryset(request, PRINCIPAL_SCOPE, Role,
```

Comment:
Good fix! This reminds me the above line 87, if scope is principal, maybe we should add this annotation too?

Reply:
Done!